src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -6,31 +6,33 @@
 
 use Friendica\BaseObject;
 use Friendica\Content\Feature;
-use Friendica\Database\DBA;
+use Friendica\Content\Text\BBCode;
+use Friendica\Content\Text\Plaintext;
+use Friendica\Core\Cache;
 use Friendica\Core\Config;
 use Friendica\Core\Logger;
 use Friendica\Core\System;
 use Friendica\Protocol\Activity;
 use Friendica\Util\HTTPSignature;
 use Friendica\Core\Protocol;
-use Friendica\Model\Conversation;
-use Friendica\Model\Contact;
+use Friendica\Core\System;
```

Comment:
`use Friendica\Core\System` cannot be used twice. :-)

Reply:
Maybe it would have been spotted easier if the use statements were in alphabetical order.

Reply:
Fixed.

Reply:
That’s what it usually is, but during some merges/rebases, these lines get duplicated. Since these operations are done outside of any IDE, it is harder to spot the issue. Thank you for your patience.